buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -32,11 +33,60 @@ fun CompileOptions.java17() {
     targetCompatibility = JavaVersion.VERSION_17
 }
 
-@Suppress("UnstableApiUsage")
-fun LibraryDefaultConfig.setLibraryVersion(
-    versionCode: Int = AndroidConfig.VERSION.code,
-    versionName: String = AndroidConfig.VERSION.name
-) {
-    buildConfigField("int", "SDK_VERSION_CODE", "$versionCode")
-    buildConfigField("String", "SDK_VERSION_NAME", "\"$versionName\"")
+fun Project.androidLibraryConfig() {
+    extensionConfig<LibraryExtension> {
+        compileSdk = AndroidConfig.TARGET_SDK
+        buildToolsVersion = AndroidConfig.BUILD_TOOLS_VERSION
+
+        defaultConfig {
+            minSdk = AndroidConfig.MIN_SDK
+            buildFeatures {
+                buildConfig = true
```

Comment:
in fact we need it only in the core module. Not sure how it affects compilation speed by enabling it for all modules.

Reply:
Yep, changing that